### DISCUSSION TITLE
Test Coverage Reports with Qodana

### DISCUSSION THREAD

**Discussion by archiegq21** [JetBrains/Qodana#126] [Q&A] [answered]
Can Qodana also generate Test Coverage Results together with its analysis?

**Comment by brichbash** [comment, marked as answer]
Hello,
It is not available yet. We do have it on our roadmap but I can't give you any ETA.
Please vote for the corresponding issue [on our tracker](https://youtrack.jetbrains.com/issue/QD-2122/Support-Code-Coverage) to get notified when there are any updates.

**Comment by archiegq21** [comment]
Looking forward for this. Just wondering as well, I assume this would integrate with [Kover](https://github.com/Kotlin/kotlinx-kover) ?

**Reply by brichbash** [reply]
It's better to leave a comment in the issue with a tool that you'd like to see. We want to support as many as possible but it's hard to tell what obstacles we'll come across.

**Comment by mhashim6** [comment]
@brichbash Hi, the youtrack ticket you listed is for code coverage, not test coverage.

Should we use the same ticket for both?
